src/rust/examples/web-server/README.md

```diff
@@ -2,4 +2,92 @@
 
 This web server example is based on the [JS server example](https://github.com/planet-nine-app/sessionless/blob/main/src/javascript/example/server/server.js).
 
-Visit the `/client` endpoint to test how it works.
+Requests, responses and underlying features might differ from the original, but the whole logic design is the same.
+
+___
+
+## Endpoint list
+- **[ANY]** `/client`
+- **[ANY]** `/register`
+- **[ANY]** `/cool-stuff`
+- **[ANY]** `/associate`
+- **[POST, GET]** `/value`
+- _­+ resources_
+
+To easily "register" and access "cool-stuff" visit `/client`.
+
+___
+
+## API Docs
+
+### ­> `/register`
+payload:
+```json
+{
+  "pub_key": <PUBLIC_KEY>,
+  "timestamp": "now",
+  "entered_text": "any"
+}
+```
+headers:
+- signature - Payload signature.
+
+### ­> `/cool-stuff`
+payload:
+```json
+{
+    "timestamp": "now",
+    "coolness": "any",
+    "uuid": <UUID>,
+}
+```
+headers:
+- signature - Payload signature.
+
+### ­> `/associate`
+payload:
+```json
+{
+    "message1": <BASE64_ENCODED_MESSAGE>,
```

Comment:
For all the messages here can you either put what the message should have in it, or give an example? Or both I guess.